curp/src/server/cmd_worker/conflict_checked_mpmc.rs

```diff
@@ -404,8 +436,13 @@ impl<C: Command> Filter<C> {
                     let v = self.get_vertex_mut(vid);
                     match v.inner {
                         VertexInner::Cmd { ref mut as_st, .. } => {
-                            debug_assert!(matches!(*as_st, AsState::NotSynced));
-                            *as_st = AsState::AfterSyncReady(index);
+                            if let AsState::NotSynced(prepare) = as_st.clone() {
+                                *as_st = AsState::AfterSyncReady(index, prepare);
+                            } else {
+                                unreachable!(
+                                    "AS State should be AsState::NotSynced but found {as_st:?}"
```

Comment:
Please replace AS with "After Sync" for that it's easier for debugging.

Reply:
Done